block_coursefeedback.php

```diff
@@ -75,6 +75,7 @@ public function get_content() {
                 && block_coursefeedbck_coursestartcheck_good($config, $this->page->course->id)) {
             // Feedback with questions is active.
             if (has_capability("block/coursefeedback:viewanswers", $context)) {
+                // For Trainer show the informative notification
```

Comment:
Wo ist der Code? Ein Kommentar verändert nichts...

Reply:
Änderung in der access.php